features/media-regenerate.feature

```diff
@@ -383,3 +383,216 @@ Feature: Regenerate WordPress attachments
       Success: Regenerated 1 of 1 images.
       """
     And the wp-content/uploads/white-150-square-150x150.jpg file should not exist
+
```

Comment:
Should we add test cases for when `--image_size=<size>` is used with `--only-missing` and `--skip-delete` ?